service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -629,4 +629,13 @@ public DynamicList buildDynamicHearingList(UUID selected) {
     private final HearingCancellationReason vacatedReason;
     private final List<ProceedingType> proceedingType;
     private final State closedStateRadioList;
+
+    private final LocalDateTime hearingEndDateConfirmation;
+    private final LocalDateTime hearingStartDateConfirmation;
+
+    @JsonIgnore
+    public boolean isHearingDateInPast() {
```

Comment:
hmm, not sure on the name as this test is really `doesHearingStartAndEndInThePast` but it's fine.

Reply:
I think it's fine as it's one or the other, is hearing start date in past or is hearing end date in past

Reply:
or can be both either